private/buf/bufformat/testdata/proto2/option/v1/option_message_field.golden.proto

```diff
@@ -48,10 +48,10 @@ message Test {
         // Trailing comment on '{'.
 
         // Leading comment on 'foo'.
-        foo: "goo"/* Leading comment on ',' */, // Trailing comment on foo
+        foo: "goo" /* Leading comment on ',' */, // Trailing comment on foo
 
         // Leading comment on extension name.
-        [/* One */foo.bar.Test.Nested._NestedNested._garblez/* Two */]/* Three */: "boo" // Trailing comment on extension name.
+        [ /* One */ foo.bar.Test.Nested._NestedNested._garblez /* Two */ ] /* Three */ : "boo" // Trailing comment on extension name.
```

Comment:
nit: This is another place where adding extra space after `[` and before `]` looks like it could be trimmed.

```suggestion
        [/* One */ foo.bar.Test.Nested._NestedNested._garblez /* Two */] /* Three */ : "boo" // Trailing comment on extension name.
```